### PR TITLE
chore(npm dep): update npm install to try sorted out dependabot prs

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

We are here only commiting the changes in the versions when we run `npm install` so that we can see if we can the dependabot fix prs run properly, see:  https://github.com/replicatedhq/kURL/security/dependabot

#### Which issue(s) this PR fixes:

Fixes # NONE

### CODE REVIEW

**Review by emosbaugh — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,943 +1,8 @@
 {
   "name": "kurl",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (rrpolanco) [inline package-lock.json:4]: Can you explain why this is changing? Doesn't NPM set this?

File: package-lock.json [hunk 2]
```diff
@@ -1,943 +1,8 @@
...
   "requires": true,
-  "packages": {
```
Comment (rrpolanco) [inline package-lock.json:6]: Is the goal to _not_ explicitly set the package versions so there aren't any conflicts with dependabot proposed changes? What's the difference between ths _packages_ section vs the package.json [_depenedencies_](https://github.com/replicatedhq/kURL/blob/1e21097e1844c1a6f91a46be60b0778075bd00e0/package.json#L9-L13) section?

File: package-lock.json (hunk 1 above)
Comment (camilamacedo86) [reply package-lock.json:4]: Yes, the changes in this PR happened by running npm install only.

**Comment by camilamacedo86** [comment]
It seems no longer required.
After we merge all bots update and I update the npm version used locally the latest version is no longer modified.

**Pull request closed without merging**